File: erettsegi_emelt/2012_may/java/Fuvar.java
public class Fuvar{
    
    public final int nap;
    public final int sorszam;
    public final int tavolsag;
    
    public Fuvar(String line) {
        var split = line.split(" ");
        
        nap = Integer.parseInt(split[0]);
        sorszam = Integer.parseInt(split[1]);
        tavolsag = Integer.parseInt(split[2]);
    }
}

File: erettsegi_emelt/2012_may/c#/Fuvar.cs
public class Fuvar {
    public readonly int nap;
    public readonly int sorszam;
    public readonly int tavolsag;

    public Fuvar(string data) {
        var split = data.Split(' ');

        nap = int.Parse(split[0]);
        sorszam = int.Parse(split[1]);
        tavolsag = int.Parse(split[2]);
    }
}